Library/Homebrew/cmd/cat.rb

```diff
@@ -25,6 +26,17 @@ def cat
     raise "`brew cat` doesn't support multiple arguments" if args.remaining.size > 1
 
     cd HOMEBREW_REPOSITORY
-    safe_system "cat", formulae.first.path, *Homebrew.args.passthrough
+    pager = "cat"
+    unless ENV["HOMEBREW_BAT"].nil?
+      name = "bat"
+      unless Formula[name].installed?
```

Comment:
I reckon it's worth just stripping out this logic entirely; people will figure it out from the command missing as otherwise we'll need to do a lot more error-handling here.

Reply:
Deleted.